core/scss/utilities/mixins/typography/_headings.scss

```diff
@@ -11,7 +11,7 @@
   clear: both;
   line-height: $heading-line-height;
 
-  @include margin(1.5em null 0.5em);
+  @include margin(0 null 0.5em);
```

Comment:
No top margins. See: https://getbootstrap.com/docs/4.0/content/reboot/#approach

This makes it so much easier to align things in grids.

Reply:
@josephgknox FYI ^^